### DISCUSSION TITLE
If I want to add the Demographic of India, How do i do so?

### DISCUSSION THREAD

**Discussion by Benjim0n** [synthetichealth/synthea#1404] [Q&A]
I know that by referring to this https://github.com/synthetichealth/synthea/wiki/Demographics-for-Other-Areas that like its possible but is there a script to input it into the geographic.csv or must we manually input it to the csv directly like here.

I would want to input maybe 3 districts of India only.

![image](https://github.com/synthetichealth/synthea/assets/82871282/5b9b9170-2a82-4375-bf95-188933df22d6)

**Comment by jawalonoski** [comment]
Yes, you either need to manually input it into a CSV file, or if you have the data in another format, write your own script to perform some type of Extract-Transform-Load to convert it. We can't provide a one-size-fits-all script because every country has its own data sources and formats.
